realm/realm-library/src/androidTest/java/io/realm/RealmAsyncQueryTests.java

```diff
@@ -1947,6 +1947,36 @@ public void onChange() {
         result.load();
     }
 
+    @Test
+    @RunTestInLooperThread
+    public void testAsyncTransactionWorksWithAsyncQuery() {
```

Comment:
I could use a comment about what this unit test is for. You are doing nothing with the `findAllAsync` result as far as I can tell?